### PR TITLE
test(typings): additional tests for typings and fixes

### PR DESCRIPTION
**Pull request by layershifter**
Ref #1621

This PR adds two features:
- adds test for `[key: string]: any;` signature in `ComponentProps`
- adds `ignoredProps` option, with can skip props from typings (like in `FormRadio.d.ts`)

Also, fixes known issues and adds tests for `Portal` and `Popup` like in a `Modal` component

### CODE REVIEW

File: test/specs/collections/Form/FormFieldCheckbox-test.js [hunk 1]
```diff
@@ -5,7 +5,9 @@ import Checkbox from 'src/modules/Checkbox/Checkbox'
 import * as common from 'test/specs/commonTests'
 
 describe('FormCheckbox', () => {
-  common.isConformant(FormCheckbox)
+  common.isConformant(FormCheckbox, {
+    ignoredProps: ['type'],
```
Comment (levithomason) [inline test/specs/collections/Form/FormFieldCheckbox-test.js]: Nit: given that the conformance test covers many things, can we rename this option to specify that it only ignores the prop in typings?

Perhaps `ignoreTypingsProps` or similar?

**Comment by levithomason** [comment]
Just one comment regarding the option name.

File: test/specs/collections/Form/FormFieldCheckbox-test.js (hunk 1 above)
Comment (layershifter) [reply test/specs/collections/Form/FormFieldCheckbox-test.js]: Yes, it's a good suggestion. I will fix it in evening

**Review by layershifter — COMMENTED** [review]
@levithomason I've updated option name, so I think that we're ready to go 🚌

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.68.3`.

**Pull request merged**